### PR TITLE
EVG-6130 Make amboy groups configurable

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: service/templates/admin.html [hunk 1]
```diff
@@ -911,6 +911,14 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 				                    <md-input-container class="control" style="width:45%; margin-left:50px;">
 										<label>Local storage size</label>
 										<input type="number" ng-model="Settings.amboy.local_storage_size">
+										<label>Group default workers</label>
```
Comment (john-m-liu) [inline service/templates/admin.html]: You've got some css problems 
image

File: config_amboy.go [hunk 2]
```diff
@@ -8,12 +8,16 @@ import (
...
+	Name                           string `bson:"name" json:"name" yaml:"name"`
+	SingleName                     string `bson:"single_name" json:"single_name" yaml:"single_name"`
+	DB                             string `bson:"database" json:"database" yaml:"database"`
+	PoolSizeLocal                  int    `bson:"pool_size_local" json:"pool_size_local" yaml:"pool_size_local"`
+	PoolSizeRemote                 int    `bson:"pool_size_remote" json:"pool_size_remote" yaml:"pool_size_remote"`
+	LocalStorage                   int    `bson:"local_storage_size" json:"local_storage_size" yaml:"local_storage_size"`
+	GroupDefaultWorkers            int    `bson:"group_default_workers" json:"group_default_workers" yaml:"group_default_workers"`
+	GroupBackgroundCreateFrequency int    `bson:"group_background_create_frequency" json:"group_background_create_frequency" yaml:"group_background_create_frequency"`
+	GroupPruneFrequency            int    `bson:"group_prune_frequency" json:"group_prune_frequency" yaml:"group_prune_frequency"`
+	GroupTTL                       int    `bson:"group_ttl" json:"group_ttl" yaml:"group_ttl"`
```
Comment (tychoish) [inline config_amboy.go]: put units here. Looks like they're minutes.

Comment (bsamek) [reply config_amboy.go]: fixed

File: service/templates/admin.html (hunk 1 above)
Comment (bsamek) [reply service/templates/admin.html]: fixed

File: config_amboy.go [hunk 3]
```diff
@@ -85,5 +93,21 @@ func (c *AmboyConfig) ValidateAndDefault() error {
 		c.LocalStorage = defaultAmboyLocalStorageSize
 	}
 
+	if c.GroupDefaultWorkers == 0 {
```
Comment (john-m-liu) [inline config_amboy.go]: Maybe validation for <1 or unreasonably large?

Comment (bsamek) [reply config_amboy.go]: I set the default if negative now. Since these are admin routes, I think maybe it's okay to allow large numberfs.

**Pull request merged**